homeassistant/components/deconz/climate.py

```diff
@@ -122,6 +146,33 @@ async def async_set_hvac_mode(self, hvac_mode: str) -> None:
 
     # Temperature control
 
+    # Preset control
+
+    @property
+    def preset_mode(self) -> Optional[str]:
+        """Return preset mode."""
+        for hass_preset_mode, preset_mode in PRESET_MODES.items():
+            if self._device.preset == preset_mode:
+                return hass_preset_mode
+
+        return None
+
+    @property
+    def preset_modes(self) -> list:
+        """Return the list of available preset modes."""
+        return list(PRESET_MODES)
+
+    async def async_set_preset_mode(self, preset_mode: str) -> None:
+        """Set new preset mode."""
+        if preset_mode not in PRESET_MODES:
+            raise ValueError(f"Unsupported preset mode {preset_mode}")
+
+        data = {"preset": PRESET_MODES[preset_mode]}
+
+        await self._device.async_set_config(data)
+
+    # Temperature control
```

Comment:
This comment is now here twice?

Reply:
Probably from the rebase, I will fix that